MIGRATION.md

```diff
@@ -0,0 +1,324 @@
+# Assumptions
+
+You are running Splunk Connect for Kubernetes(SCK) 1.4.9 and want to migrate to Splunk Connect for Kubernetes Opentelemetry(SCK-OTEL) 1.0 GA. Previous versions of SCK are out of scope, but this guide is applicable for previous versions. You might encounter some version specific issues (such as missing/changed config options) but that shouldn't prevent you from proceeding with migration.
```

Comment:
I believe the product is called _Splunk OpenTelemetry Connector for Kubernetes_ not _Splunk Connect for Kubernetes Opentelemetry(SCK-OTEL)_. Please update naming in this PR

Reply:
updated